bears/haml/HAMLLintBear.py

```diff
@@ -0,0 +1,396 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.GemRequirement import GemRequirement
+import yaml
+
+
+@linter(executable='haml-lint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+) \[(?P<severity>W|E)\] '
+                     r'(?P<message>.*)',
+        severity_map={'E': RESULT_SEVERITY.MAJOR,
+                      'W': RESULT_SEVERITY.NORMAL})
```

Comment:
This severity map is part of the default map :)
(It's actually case insensitive, filling an issue to document that)

Reply:
So you can just omit the parameter :+1:

Reply:
Hey @Makman2 could you please explain what exactly do I need to change here? 😄

Reply:
just leave `severity_map` altogether ;)